UnityProject/Assets/Scripts/Core/Chat/Chat.Process.cs

```diff
@@ -456,18 +466,20 @@ private static string GetChannelColor(ChatChannel channel)
 		if (channel.HasFlag(ChatChannel.Local)) return ColorUtility.ToHtmlStringRGBA(Instance.localColor);
 		if (channel.HasFlag(ChatChannel.Combat)) return ColorUtility.ToHtmlStringRGBA(Instance.combatColor);
 		if (channel.HasFlag(ChatChannel.Warning)) return ColorUtility.ToHtmlStringRGBA(Instance.warningColor);
-		return ColorUtility.ToHtmlStringRGBA(Instance.defaultColor); ;
+		return ColorUtility.ToHtmlStringRGBA(Instance.defaultColor);
+		;
```

Comment:
Codacy found an issue: [Remove this empty statement.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6319358)